README.md

```diff
@@ -1,37 +1,56 @@
-# Table of Contents
+# nyaml
 
+## Scope and purpose:
+## may edit again
+Make working with the NeXus Schema Definition Language more convenient for end users.
+NeXus and its respective NeXus Definition Language (NXDL) represents a concerted effort for standardizing the terminology and granularization for the exchange of serialized data and metadata within and across scientific communities. NeXus is rooted in the neutron, X-ray, and muon research [J. Appl. Cryst. (2015). 48, 301-305](https://doi.org/10.1107/S1600576714027575). The tool nyaml is an effort of members of the solid-state physics community within the German National Research Data Infrastructure ([German NFDI](https://www.github.com/FAIRmat-NFDI) to extend NeXus for standardized information exchange in the research fields of materials characterization.
+
+NeXus describes concepts through general data storage objects (so-called base classes). From these building blocks, a so-called application definition is composed. This is a measurement- and instrument-specific graph that can be used to define which pieces of information are communicated with instance data such as files or database artifacts. Base classes and application definitions are defined through a so-called NXDL schema definition file using the Extensible Markup Language, [XML](https://www.w3.org/TR/REC-xml/REC-xml-20081126.xml). The nyaml tool makes the process of working and editing NXDL schema definitions more efficient by using Yet Another Markup Language ([YAML](https://yaml.org/) with its indentation-driven approach to eliminate the need for editing starting and ending XML tags. Thereby, the schema definitions read more concisely and enable grasping more intuitively class inheritance that NeXus allows for. 
```

Comment:
I like the idea of introducing NeXus a bit earlier than what was in the initial draft. However, here there are some things removed/shortened which should still be there IMO, so we should aim to merge the two paragraphs here and in the original draft.

Reply:
Happy for suggestions, the introductory part was mostly a copy of content from a NIAC side. I see value in this being shorter. Indeed, when writing the documentation for these tools I would rather test this time the idea to do it less verbose but to the point